index.js

```diff
@@ -23,16 +27,31 @@ const portCheckSequence = function * (ports) {
 
 module.exports = async options => {
 	let ports = null;
-
+	const sweep = 1000 * 15;
```

Comment:
I think the variable should be more descriptive and also add a code comment. And move it to the top-level.